packages/vite/package.json

```diff
@@ -88,7 +88,7 @@
     "esbuild": "^0.21.3",
     "postcss": "^8.4.39",
     "rollup": "^4.13.0",
-    "rolldown": "^0.10.5"
+    "rolldown": "/Users/likui/Github/rolldown/packages/rolldown"
```

Comment:
You should not use the local absolute path here.

Reply:
Yeah. I know it. It will be replaced for nightly release.